Cannot read property '0' of undefined + Captive Portal not working

```
Connman initialized                                                                                                                                   
WiFi initialized                                                                                                                                      
Mon, 14 Sep 2015 09:04:10 GMT body-parser deprecated bodyParser: use individual json/urlencoded middlewares at app.js:148:15                          
Mon, 14 Sep 2015 09:04:10 GMT body-parser deprecated undefined extended: provide extended option at ../node_modules/body-parser/index.js:105:29       
Trying to join previously known networks                                                                                                              
Getting networks list                                                                                                                                 
Hotspot enabled                                                                                                                                       
                                                                                                                                                      
/usr/src/app/node_modules/bluebird/js/main/async.js:43                                                                                                
        fn = function () { throw arg; };                                                                                                              
                                 ^                                                                                                                    
TypeError: Cannot read property '0' of undefined                                                                                                      
    at /usr/src/app/src/app.js:57:45                                                                                                                  
    at tryCatcher (/usr/src/app/node_modules/bluebird/js/main/util.js:26:23)                                                                          
    at Function.Promise.attempt.Promise.try (/usr/src/app/node_modules/bluebird/js/main/method.js:31:24)                                              
    at async.retry.callback.table (/usr/src/app/src/app.js:56:28)                                                                                     
    at /usr/src/app/node_modules/async/lib/async.js:622:21                                                                                            
    at /usr/src/app/node_modules/async/lib/async.js:689:13                                                                                            
    at iterate (/usr/src/app/node_modules/async/lib/async.js:265:13)                                                                                  
    at /usr/src/app/node_modules/async/lib/async.js:277:29                                                                                            
    at /usr/src/app/node_modules/async/lib/async.js:44:16                                                                                             
    at /usr/src/app/node_modules/async/lib/async.js:694:17                                                                                            
    at /usr/src/app/node_modules/async/lib/async.js:173:37                                                                                            
    at null._onTimeout (/usr/src/app/node_modules/async/lib/async.js:631:25)                                                                          
    at Timer.listOnTimeout [as ontimeout] (timers.js:112:15)  

npm ERR! Linux 3.18.5                                                                                                                                 
npm ERR! argv "node" "/usr/local/bin/npm" "start"                                                                                                     
npm ERR! node v0.10.38                                                                                                                                
npm ERR! npm  v2.10.1                                                                                                                                 
npm ERR! code ELIFECYCLE                                                                                                                              
npm ERR! resin-wifi-connect@1.0.0 start: `cd src && node app.js`                                                                                      
npm ERR! Exit status 8                                                                                                                                
npm ERR!                                                                                                                                              
npm ERR! Failed at the resin-wifi-connect@1.0.0 start script 'cd src && node app.js'.                                                                 
npm ERR! This is most likely a problem with the resin-wifi-connect package,                                                                           
npm ERR! not with npm itself.                                                                                                                         
npm ERR! Tell the author that this fails on your system:                                                                                              
npm ERR!     cd src && node app.js                                                                                                                    
npm ERR! You can get their info via:                                                                                                                  
npm ERR!     npm owner ls resin-wifi-connect                                                                                                          
npm ERR! There is likely additional logging output above.                                                                                             
                                                                                                                                                      
npm ERR! Please include the following file with any support request:                                                                                  
npm ERR!     /usr/src/app/npm-debug.log   
```

--- a/src/app.coffee
+++ b/src/app.coffee
@@ -28,7 +28,7 @@ catch
 ignore = ->
 
 getIptablesRules = (callback) ->
-	async.retry {times: 100, interval: 100}, (cb) ->
+	async.retry {times: 100, interval: 200}, (cb) ->
 		Promise.try ->
 			os.networkInterfaces().tether[0].address
 		.nodeify(cb)

